src/Grpc.AspNetCore.Server/Internal/PipeExtensions.cs

```diff
@@ -361,11 +361,11 @@ public static async ValueTask<T> ReadSingleMessageAsync<T>(this PipeReader input
             }
         }
 
-        private static bool TryReadMessage(ref ReadOnlySequence<byte> buffer, HttpContextServerCallContext context, [NotNullWhen(true)] out ReadOnlySequence<byte>? message)
+        private static bool TryReadMessage(ref ReadOnlySequence<byte> buffer, HttpContextServerCallContext context, [NotNullWhen(true)] out ReadOnlySequence<byte> message)
```

Comment:
```suggestion
        private static bool TryReadMessage(ref ReadOnlySequence<byte> buffer, HttpContextServerCallContext context, out ReadOnlySequence<byte> message)
```
a) valuetype
b) nullability got removed, so it's always non-null